doc/additional_commands.rst

```diff
@@ -0,0 +1,100 @@
+Universum modes
+===============
+
+Along with default mode Universum provides the following useful additional commands:
+
+* `init <additional_commandst#init>`_
+* `run <additional_commandst#run>`_
+* `poll <additional_commandst#poll>`_
+* `submit <additional_commandst#submit>`_
+* `github-handler <additional_commandst#github-handler>`_
+
+
+.. _additional_commandst#init:
+
+Initialize Universum
+--------------------
+
+If you have Universum installed via ``{pip} install -U universum`` or downloaded it from GitHub and changed
+working directory to project root, run ``{python} -m universum init`` to create an example configuration file,
+that can be :doc:`updated according to your project needs <configuring>` later.
+
+Running this command will not only create a configuration file, but will also provide you with a command line to
+execute it with Universum.
+
+
+.. _additional_commandst#run:
+
+Run Universum locally (Non-CI mode)
+-----------------------------------
+
+The ``{python} -m universum run`` subcommand uses provided :doc:`Universum config <configuring>` to execute
+the set up scenario locally. This allows to easily perform all required checks *before* submitting a change
+to VCS. This mode can generally be used as a wrapper for a complex build system.
+
+Unlike default `Universum` mode, using ``{python} -m universum run``:
+
+* does not require most of :doc:`build parameters <args>` - especially VCS-related ones
+* does not :doc:`report build results <code_report>` to any code review system
+* works with sources 'in place', without copying
+* :ref:`cleans artifact folder <clean_artifacts>` before build automatically.
```

Comment:
1. "automatically" must be the first word of this item
2. add "the" 
"automatically cleans the artifact folder before the build".